src/components/Table/GeneralTableHeaderCell.tsx

```diff
@@ -71,7 +73,7 @@ export default function GeneralTableHeaderCell({
   const headerContent = tooltip ? (
     <Stack
       direction="row"
-      spacing={0.2}
+      spacing={isTableTooltip ? 0.2 : 1}
```

Comment:
`TableTooltip` that requires a click and regular tooltips have different spacing, layout needs.